clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp

```diff
@@ -102,12 +102,13 @@ class UncountedCallArgsChecker
         // if ((*P)->hasAttr<SafeRefCntblRawPtrAttr>())
         //  continue;
 
-        const auto *ArgType = (*P)->getType().getTypePtrOrNull();
-        if (!ArgType)
+        QualType ArgType = (*P)->getType().getCanonicalType();
+        const auto *TypePtr = ArgType.getTypePtrOrNull();
```

Comment:
`getTypePtrOrNull()` is unnecessary most of the time because you can do all the same operations on `QualType` directly, thanks to the overloaded `QualType::operator->()`.

I think `isUncountedPtr()` should simply accept a `QualType` directly.

Reply:
Oh, I see. Let me make that change in a separate PR.

Reply:
https://github.com/llvm/llvm-project/pull/110213